<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminModifyBookDetail</class>
 <widget class="QWidget" name="AdminModifyBookDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_bookposition">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>380</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>图书位置</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_ISBN">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>220</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>ISBN</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_publish">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>出版社</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_remain">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>340</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>总库存数</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_intro">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>480</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>简介</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_author">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>140</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>作者</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_publishtime">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>300</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>出版时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_classify">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>分类</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bookphoto">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>200</width>
     <height>330</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bookname">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>书名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_bookname">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>100</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_author">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>140</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_publish">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>180</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_ISBN">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>220</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_classify">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>260</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_publishtime">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>300</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_remain">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>340</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_bookposition">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>380</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="book_pushButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>810</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_modifyimage">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>430</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>修改图片</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>550</y>
     <width>1050</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
